### PR TITLE
Update 19.02.md

### PR DESCRIPTION
**Pull request by mariano-capurro**
test test test

### CODE REVIEW

File: android/19.02.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <p>
   This document will guide you through the process of Countly SDK installation.
 </p>
+<p>
+	Test test test
+	</p>
```
Comment (ArtursKadikis) [inline android/19.02.md:6]: Please fix tab spacing

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]
Please add another tag

**Comment by ArtursKadikis** [comment]
Nevermind the tag thing fix the tab spacing

**Comment by mariano-capurro** [comment]
I don't agree with you at all

**Comment by mariano-capurro** [comment]
> Please add another tag
sure

File: android/19.02.md (hunk 1 above)
Comment (mariano-capurro) [reply android/19.02.md:6]: Alright done. Please check now

**Pull request merged**